mlrun/projects/project.py

```diff
@@ -3846,6 +3846,18 @@ def _init_function_from_dict(
                 tag=tag,
             )
 
+    elif image and kind in mlrun.runtimes.RuntimeKinds.nuclio_runtimes():
```

Comment:
I'm missing a validation that `image` exists if the runtime kind is `application`. we can't guess what image to run in the sidecar...

Reply:
discussed offline